src/test/java/org/broadinstitute/hellbender/tools/walkers/realignmentfilter/RealignmentEngineUnitTest.java

```diff
@@ -0,0 +1,123 @@
+package org.broadinstitute.hellbender.tools.walkers.realignmentfilter;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import org.apache.commons.math3.util.Pair;
+import org.broadinstitute.hellbender.utils.bwa.BwaMemAlignment;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+public class RealignmentEngineUnitTest {
+    private static final SAMFileHeader HEADER = ArtificialReadUtils.createArtificialSamHeader();
+    private static final int REF_INDEX = 0;
+    private static final String SOURCE = "source";
+    private static final String CONTIG = "chr1";
+    private static final int TOL = 1;
+
+    @Test
+    public void testSupportsVariant() {
+        final GATKRead refRead = makeRead("refRead", 1, "ACGTACGT", "8M");
```

Comment:
`refRead` is not used, but I think we should keep the ref sequence (extended to include "GGGGG") somehow, either as a comment or as an unused variable

Reply:
I had meant to make sure that the ref read doesn't support any of the variants and forgot to do so.  Fixed.